docs/events/syncerror.md

```diff
@@ -12,7 +12,7 @@ Unlike most of FHIRcast events, `syncerror` is an infrastructural event and does
 
 Key | Optionality | Fhir operation to generate context | Description
 ----- | -------- | ---- | ---- 
-`operationoutcome` | OPTIONAL | `OperationOutcome` | FHIR resource describing an outcome of an unsuccessful system action. The OperationOutcome SHALL use a code of processing. The OperationOutcome's `details.coding.code` SHALL contain the id of the event that this error is related to as a `code` with the `system` value of "https://fhircast.hl7.org/events/syncerror/eventid". Other `coding` values can be included with different `system` values so as to include extra information about the `syncerror`.
+`operationoutcome` | OPTIONAL | `OperationOutcome` | FHIR resource describing an outcome of an unsuccessful system action. The OperationOutcome SHALL use a code of processing. The OperationOutcome's `details.coding.code` SHALL contain the id of the event that this error is related to as a `code` with the `system` value of "https://fhircast.hl7.org/events/syncerror/eventid" and the name of the error'd event with a `system` value of "https://fhircast.hl7.org/events/syncerror/eventname" . Other `coding` values can be included with different `system` values so as to include extra information about the `syncerror`.
```

Comment:
Few things:
- Should processing be in the fancy quotes `` since it is a code value or is it just a string (in which case should it be in normal quotes)?
- Is "error'd" correct? Should it be like the previous sentence and be "event that this error is related to"? That's wordy so I am open to a shorter wording.

Reply:
updated. What do you think?